docs/docs/react-without-jsx.md

```diff
@@ -0,0 +1,50 @@
+---
+id: react-without-jsx
+title: React Without JSX
+permalink: docs/react-without-jsx.html
+---
+
+JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up transpilation in your build environment.
+
+Each JSX element is just syntactic sugar for calling `React.createElement(component, props, children)`. So, anything you can do with JSX can also be done with just plain JavaScript.
+
+For example, this code written with JSX:
+
+```js
+class Hello extends React.Component {
+  render() {
+    return <div>Hello {this.props.toWhat}</div>;
+  }
+}
+
+ReactDOM.render(<Hello toWhat={'World'} />,
+                document.getElementById('root'));
+```
+
+can also be written without JSX:
+
+```js
+class Hello extends React.Component {
```

Comment:
I know you show it right below, but can we do `const e =` in every example except when we're specifically discussing what JSX compiles to? It's the recommended way to use createElement.